client/src/PaginatorContext.js

```diff
@@ -30,7 +75,9 @@ const useJobPaginator = (initialCategory, initialJob) => {
             }
         });
     };
-
+    const setOrder = (order) => dispatch({ type: "UPDATE_ORDER", payload: order });
```

Comment:
Just a style question: you implemented setOrder and setMagicLabelCache as one-liners, but all the subsequent setters are formatted to take up several lines. Any reason for not using the same approach everywhere? Might be worth picking one or the other and being consistent.

Reply:
I think ill do it in several lines.